tests/unit/LicenseDropdown.spec.js

```diff
@@ -0,0 +1,68 @@
+import { mount, createLocalVue, config } from "@vue/test-utils";
```

Comment:
This file should be in the 'tests/unit/components/' folder.

Reply:
Ok, I'll put all the 3 test files created by me viz. FirstStep.spec.js , LicenseDropdown.spec.js and Step.spec.js into "/unit/specs/components"